src/Pages/SignUp/SignUp.scss

```
@@ -0,0 +1,112 @@
+@import '../../Styles/common.scss';
+
+.signUp {
+  width: 100%;
+  height: 150vh;
+  margin-top: 100px;
+  padding: 100px;
+
+  .signUpTitle {
+    text-align: center;
+    padding: 40px 0px 90px;
+    h2 {
+      font-size: 42px;
+    }
+
+    div {
+      padding: 30px 0px 0px;
+
+      span {
+        font-size: 16px;
+        font-weight: 600;
+      }
+
+      span:nth-child(1),
+      span:nth-child(2),
+      span:nth-child(4),
+      span:nth-child(5) {
+        color: #a9a99a;
+      }
+
+      span:nth-child(2),
+      span:nth-child(4) {
+        margin: 12px;
+      }
+    }
+  }
+  .signForm {
+    .signFormInfo {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      padding: 0px 0px 15px;
+      border-bottom: 1px solid black;
+      h3 {
```

```suggestion
      border-bottom: 1px solid black;
      
      h3 {
```
새로운 셀렉터 앞에서는 무조건 한줄 띄워주세요

---

감사합니다👍